components/NavigationDrawer/src/private/MDCBottomDrawerContainerViewController.m

```diff
@@ -596,33 +629,29 @@ - (void)viewDidDisappear:(BOOL)animated {
 
 - (void)preferredContentSizeDidChangeForChildContentContainer:(id<UIContentContainer>)container {
   [super preferredContentSizeDidChangeForChildContentContainer:container];
-  if ([container isKindOfClass:[UIViewController class]]) {
```

Comment:
is there a reason why we removed this conditional? See https://github.com/material-components/material-components-ios/pull/5928/files#diff-164176fc4563ef4ed712d129906eae0aR495 for more context. @codeman7 can you please take a look and see if this change is ok?